improver_tests/calibration/ensemble_calibration/test_ApplyEMOS.py

```diff
@@ -265,6 +267,69 @@ def test_land_sea_mask(self):
         )
         self.assertArrayAlmostEqual(result.data[0], expected_data_slice)
 
+    def test_null_percentiles_truncnorm_standard_shape_parameters(self):
+        """Test effect of "neutral" emos coefficients in percentile space
+        (this is small but non-zero due to limited sampling of the
+        distribution) for the truncated normal distribution."""
+        coefficients = iris.cube.CubeList([])
+        for cube in self.coefficients:
+            cube.attributes["distribution"] = "truncnorm"
+            cube.attributes["shape_parameters"] = np.array([0, np.inf], np.float32)
+            coefficients.append(cube)
+
+        expected_mean = np.mean(self.percentiles.data)
+        expected_data = np.array(
+            [
+                np.full((3, 3), 10.265101),
+                np.full((3, 3), 10.4),
+                np.full((3, 3), 10.534898),
+            ]
+        )
+        result = ApplyEMOS()(self.percentiles, coefficients, realizations_count=3)
+        self.assertIn("percentile", get_dim_coord_names(result))
+        self.assertArrayAlmostEqual(result.data, expected_data)
+        self.assertAlmostEqual(np.mean(result.data), expected_mean)
+
+    def test_null_percentiles_truncnorm_alternative_shape_parameters(self):
+        """Test effect of "neutral" emos coefficients in percentile space
+        (this is small but non-zero due to limited sampling of the
+        distribution) for the truncated normal distribution with alternative
+        shape parameters to show the truncnorm distribution having an effect."""
+        coefficients = iris.cube.CubeList([])
+        for cube in self.coefficients:
+            cube.attributes["distribution"] = "truncnorm"
+            cube.attributes["shape_parameters"] = np.array([10, np.inf], np.float32)
+            coefficients.append(cube)
+
+        expected_mean = np.mean(self.percentiles.data)
+        expected_data = np.array(
+            [
+                np.full((3, 3), 10.275656),
+                np.full((3, 3), 10.405704),
+                np.full((3, 3), 10.5385),
+            ]
+        )
+        result = ApplyEMOS()(self.percentiles, coefficients, realizations_count=3)
+        self.assertIn("percentile", get_dim_coord_names(result))
+        self.assertArrayAlmostEqual(result.data, expected_data)
+        self.assertNotAlmostEqual(np.mean(result.data), expected_mean)
```

Comment:
The two tests above are unrelated to these changes right? Just additional coverage? If not, I may need you to explain to me how these are related to the changes reviewed here.

Reply:
The main reason to add these tests was so that there was a test of the `ApplyEMOS` plugin where the `shape_parameters` attribute was set on the coefficients cube, in addition to the `distribution` attribute. I added the `test_null_percentiles_truncnorm_alternative_shape_parameters` test just check that the `truncnorm` distribution was impacting the expected data.